fe/fe-core/src/main/java/org/apache/doris/datasource/property/constants/PaimonProperties.java

```diff
@@ -0,0 +1,26 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.datasource.property.constants;
+
+public class PaimonProperties {
+    public static final String PAIMON_WAREHOUSE = "paimon.warehouse";
+    public static final String URI = "uri";
+    public static final String METASTORE = "metastore";
+    public static final String METASTORE_HIVE = "hive";
+    public static final String WAREHOUSE = "warehouse";
```

Comment:
about the PaimonProperties, I think we should not give the paimon directly, use paimon.xxx for external